Find a functionally equivalent code:android.transition.TransitionSet.createAnimators:COMMENT
Method Modifier: protected   hidden      
Comment:/**
 * @hide
 */

@@ -1,5 +1,17 @@
 {
-    for (Transition childTransition : mTransitions) {
-        childTransition.createAnimators(sceneRoot, startValues, endValues);
+    long startDelay = getStartDelay();
+    int numTransitions = mTransitions.size();
+    for (int i = 0; i < numTransitions; i++) {
+        Transition childTransition = mTransitions.get(i);
+        // the transitions sequentially.
+        if (startDelay > 0 && (mPlayTogether || i == 0)) {
+            long childStartDelay = childTransition.getStartDelay();
+            if (childStartDelay > 0) {
+                childTransition.setStartDelay(startDelay + childStartDelay);
+            } else {
+                childTransition.setStartDelay(startDelay);
+            }
+        }
+        childTransition.createAnimators(sceneRoot, startValues, endValues, startValuesList, endValuesList);
     }
 }

